### PR TITLE
Feature/122 add ability to use layered concentric infinite cylinders with refractive index mismatch

### PR DESCRIPTION
**Pull request by hayakawa16**
Hi, this is a *draft* of a PR so I can see file differences and make sure all looks good.  I will assign reviewers when I'm ready for review.

### CODE REVIEW

**Comment by hayakawa16** [comment]
I'm ready for review.

**Comment by hayakawa16** [comment]
I just saw sonarcloud review.  Give me a bit to address these.

**Comment by hayakawa16** [comment]
Okay I'm ready for review.

**Review by janakarana — APPROVED** [review]
Everything looks fine to me

File: src/Vts.Test/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidationTests.cs
Comment (lmalenfant) [inline src/Vts.Test/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidationTests.cs:51]: This is nitpicky but I like to see a space between methods and therefore a line feed after the method and before the comments for the next method. I use space to help me find the individual methods.

File: src/Vts.Test/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidationTests.cs
Comment (lmalenfant) [inline src/Vts.Test/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidationTests.cs:89]: Line feed

File: src/Vts.Test/MonteCarlo/Tissues/MultiConcentricInclusionTissueTests.cs
Comment (lmalenfant) [inline src/Vts.Test/MonteCarlo/Tissues/MultiConcentricInclusionTissueTests.cs:116]: Line feed

File: src/Vts.Test/MonteCarlo/Tissues/MultiConcentricInclusionTissueTests.cs
Comment (lmalenfant) [inline src/Vts.Test/MonteCarlo/Tissues/MultiConcentricInclusionTissueTests.cs:284]: Extra line feed

File: src/Vts.Test/MonteCarlo/Tissues/SingleInfiniteCylinderTissueTests.cs
Comment (lmalenfant) [inline src/Vts.Test/MonteCarlo/Tissues/SingleInfiniteCylinderTissueTests.cs:32]: Line feed

File: src/Vts.Test/MonteCarlo/Tissues/SingleInfiniteCylinderTissueTests.cs
Comment (lmalenfant) [inline src/Vts.Test/MonteCarlo/Tissues/SingleInfiniteCylinderTissueTests.cs:75]: Line feed

File: src/Vts.Test/MonteCarlo/Tissues/SingleInfiniteCylinderTissueTests.cs
Comment (lmalenfant) [inline src/Vts.Test/MonteCarlo/Tissues/SingleInfiniteCylinderTissueTests.cs:104]: Line feed

File: src/Vts/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidation.cs
Comment (lmalenfant) [inline src/Vts/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidation.cs:62]: We don't need the braces here if it is on one line.

File: src/Vts/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidation.cs
Comment (lmalenfant) [inline src/Vts/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidation.cs:81]: We don't need the braces if the code is on one line.

File: src/Vts/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidation.cs
Comment (lmalenfant) [inline src/Vts/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidation.cs:39]: We don't need the braces if the code is on one line.

File: src/Vts/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidation.cs
Comment (lmalenfant) [inline src/Vts/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidation.cs:48]: We don't need the braces if the code is on one line.

**Review by lmalenfant — COMMENTED** [review]
Most of these are just suggestions so I have no problem approving if you don't want to fix them now.

**Comment by hayakawa16** [comment]
Hi @lmalenfant, thanks for your review. I can make these edits.

**Comment by hayakawa16** [comment]
I have updated code and pushed.

On another topic.  I pulled master and ran an infile that I ran in August 2023 and obtained identical results.  Then I pulled this branch and ran the same infile.  The results are not identical. I think they are statistically equivalent but not sure.  I'd like to know for sure before I merge this branch into master.

**Comment by lmalenfant** [comment]
I also noticed that we cannot merge right now because I made the master branch read only. We either have to force push or remove the lock. I checked that box so we can't push to master without a PR. I need to do more research to figure out the best protection.

**Comment by hayakawa16** [comment]
I've performed my analysis of results using master and this branch and they agree within variance.

**Comment by hayakawa16** [comment]
I feel okay about merging this into master.  Should I check "Merge without waiting for requirements to be met (bypass branch protections)" and click "Merge pull request"?

**Review by lmalenfant — APPROVED** [review]

**Comment by lmalenfant** [comment]
@hayakawa16 yes go ahead and merge, I think you just need to check the box and then the button will be available.

**Pull request merged**
